Library/Homebrew/os/mac/diagnostic.rb

```diff
@@ -0,0 +1,111 @@
+module Homebrew
+  module Diagnostic
```

Comment:
This code should go in the existing file `extend/os/mac/diagnostic.rb` in `Homebrew::Diagnostic::Checks`.

Reply:
all of it?

Reply:
Yeah, I believe so. `brew diagnostic` checks live in…
```
diagnostic.rb
extend/os/linux/diagnostic.rb
extend/os/mac/diagnostic.rb
```